docs/how-to/ChIP_seq_genomic_ranges_tutorial.ipynb

```diff
@@ -0,0 +1,2733 @@
+{
```

Comment:
<p>It would be nice to link to the docs for read_table here, like:</p><p> </p>

<pre class="ql-syntax" spellcheck="false">{func}<code>~bioframe.read_table</code>
</pre><p>You'll also need to add this line to the <code>intersphinx_mapping</code> in <code>docs/conf.py</code> :</p>

<pre class="ql-syntax" spellcheck="false">    "bioframe": ("https://bioframe.readthedocs.io/en/stable/", None),  
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/genomic-features/pull/65/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>